pkgs/top-level/python-packages.nix

```diff
@@ -4395,7 +4395,9 @@ in {
 
   nbconflux = callPackage ../development/python-modules/nbconflux { };
 
-  nbconvert = callPackage ../development/python-modules/nbconvert { };
+  nbconvert = callPackage ../development/python-modules/nbconvert {
+    nbclient = self.nbclient.override { doCheck = false; };
```

Comment:
Could do with the "avoid recursion" comment left here but :shrug:

Reply:
I think `nbclient` needs to have it's tests disabled always, as discussed in https://github.com/NixOS/nixpkgs/pull/115303#discussion_r589514942 .